railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -32,6 +32,15 @@ RUN curl -sL https://github.com/nodenv/node-build/archive/master.tar.gz | tar xz
     rm -rf /tmp/node-build-master
 
 <% end -%>
+
+<% if using_bun? -%>
+ARG BUN_VERSION=<%= dockerfile_bun_version %>
+ENV BUN_INSTALL=/usr/local/bun
+ENV PATH=/usr/local/bun/bin:$PATH
+RUN curl -fsSL https://bun.sh/install | bash -s -- "${BUN_VERSION}"
```

Comment:
hey @terracatta 👋 

this is awesome! 

thanks for this PR. Just one question. does this work when `latest` is applied? 
I was also changing the dockerfile and notice this to specify the version BUT I don't think the `latest` (which is the default for dockerfile_bun_version when no bun is configured on your machine) works.

See:
```
curl -fsSL https://bun.sh/install | bash -s -- latest
curl: (22) The requested URL returned error: 404

error: Failed to download bun from "https://github.com/oven-sh/bun/releases/download/latest/bun-linux-x64.zip"
```

what I did instead was always install the latest. I found this comment https://github.com/oven-sh/bun/pull/848#issuecomment-1193697983 from @Jarred-Sumner which enforces always use the latest. so I changed this to 
``
RUN curl -fsSL https://bun.sh/install | bash
```
on mine.

wdyt?

Reply:
Done.